<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>1700</width>
     <height>900</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>792</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/device.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>-1</y>
      <width>741</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{background: #F7F7F7;
border-radius: 20px;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>105</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 80px;
height: 40px;
color: #333333;
line-height: 60px;</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>720</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>34</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 720px;
height: 110px;
background: #FFFFFF;
border-radius: 55px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>JCR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>720</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>34</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 720px;
height: 110px;
background: #FFFFFF;
border-radius: 55px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>123456</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>720</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#07A8E7;
border-radius: 20px;
color: #FFFFFF;
line-height: 54px;</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="password_CheckBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>580</y>
       <width>161</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 144px;
height: 36px;
color: #666666;
line-height: 54px;
</string>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="userName_Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F47E4B;</string>
     </property>
     <property name="text">
      <string>用户名输入错误</string>
     </property>
    </widget>
    <widget class="QLabel" name="userPassword_Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>502</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F47E4B;</string>
     </property>
     <property name="text">
      <string>密码输入错误</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
